### ISSUE TITLE
Create a new rule for powershell alias injection

### ISSUE THREAD

**Issue by lifeforms** [coreruleset/coreruleset#2688]
### Description

We detect powershell commands. However, we are not detecting shorthand aliases.

Many commands are very verbose, but they have short aliases. The Powershell function Get-Alias will show the default defined aliases. There are a lot. Most seem uncommon words fortunately. By the way, pretty funny that they aliased `curl` to `Invoke-WebRequest`. But we should triage them and balance the risk of false positives with complete coverage.

```
PS C:\Users\IEUser> get-alias

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Alias           % -> ForEach-Object
Alias           ? -> Where-Object
Alias           ac -> Add-Content
Alias           asnp -> Add-PSSnapin
Alias           cat -> Get-Content
Alias           cd -> Set-Location
Alias           CFS -> ConvertFrom-String                          3.1.0.0    Microsoft.PowerShell.Utility
Alias           chdir -> Set-Location
Alias           clc -> Clear-Content
Alias           clear -> Clear-Host
Alias           clhy -> Clear-History
Alias           cli -> Clear-Item
Alias           clp -> Clear-ItemProperty
Alias           cls -> Clear-Host
Alias           clv -> Clear-Variable
Alias           cnsn -> Connect-PSSession
Alias           compare -> Compare-Object
Alias           copy -> Copy-Item
Alias           cp -> Copy-Item
Alias           cpi -> Copy-Item
Alias           cpp -> Copy-ItemProperty
Alias           curl -> Invoke-WebRequest
...
```

Adding the shorthand aliases to the existing Powershell pm rule has a big chance of causing false positives.

It is better to create a new rule for the aliases, where the commands are preceded by a command separator (|, ; & etc) to detect them as powershell injections, and check for spacing and parameters afterwards. That would have to be a regexp rule.

Example:

`/image.jpg|cp C:\Windows\calc.exe .`

**Comment by fzipi** [comment]
@lifeforms What do you think on the 932125 ruleset?

**Comment by fzipi** [comment]
@lifeforms ping again.

**Comment by fzipi** [comment]
@lifeforms ping ping?

**Comment by Xhoenix** [comment]
The `regex-assembly/932125.ra` needs to be updated. I checked following the instructions mentioned in the file, and a few new additions to those aliases had been made since last commit.

**Comment by fzipi** [comment]
Closing this one as done.
